src/services/Functional/User.js

```diff
@@ -13,7 +13,7 @@ export default {
       ) {
         store.dispatch("auth/unsetAccessToken");
         store.dispatch("auth/setReAuthenticationState", false);
-        return Promise.resolve(true);
+        return Promise.reject("Auto Logout!");
```

Comment:
<img width="570" alt="Screenshot 2021-10-30 at 4 23 00 PM" src="https://user-images.githubusercontent.com/13185699/139529977-06cb8b20-0127-4ab9-b654-5a85f6ef409b.png">
I was trying to replicate this situation locally by intentionally setting isReAuthenticating to true and triggering the reAuthenticate method. I am getting this error - after the "Auto Logout" statement - as well!

Reply:
- Haan ye second error will be resolved by that websocket waali PR that is open
- How did you try to replicate the situation? Because whenever I'm doing I'm never getting this
- Try once again, I've made some more fixes to handle errors better. Pehle the errors which were being originated at point A were lost somewhere in between. They were not still not being `caught` properly. (hence the red exceptions). I've tried to catch them
- Also, found a few more bugs